### ISSUE TITLE
Screen is zoomed and cropped on all systems

### ISSUE THREAD

**Issue by tymime** [higan-emu/ares#360]
I think this screenshot is pretty self-explanatory:
![Screen Shot 2022-02-17 at 5 14 34 AM](https://user-images.githubusercontent.com/18604835/154489419-308bb773-fbb0-4af4-9505-b93e188ed2c3.png)
I'm on a Mac with Sierra, OpenGL driver. Output is set to "Center" and nothing else.

**Comment by tymime** [comment]
Hello??

**Comment by LukeUsher** [comment]
> Hello??

Nobody else can reproduce this issue so far; we have tried with multiple macs in multiple different configurations; can you give more details?

Are you using DPI scaling, etc?

**Comment by tymime** [comment]
Do you mean the DPI of my Mac? If yes, then my scaling options are the default for a 4K Retina monitor. The only thing I've changed is the color temperature, but I'm assuming that wouldn't affect Ares at all.
If you mean the the size settings on Ares, it looks like this on all of them- 2x, 3x, and 4x. It also looks like this on every other video setting too.

**Comment by tymime** [comment]
Any progress on this?

**Comment by Kawa-oneechan** [comment]
Apparently not.

**Comment by tymime** [comment]
On a whim, I just temporarily switched the resolution on my Mac to 1080p, and discovered that doing so corrects the zoomed-in screen. So yes, it definitely does have something to do with me having a 4K Retina monitor.
I hope you can look into this soon- I'd hate to do this every time I want to run Ares.

**Comment by jcm93** [comment]
I don't think we have this issue any longer on either the Metal or the OpenGL driver. I'd encourage you to re-test it and if it's still occurring, try to provide detailed steps.

**Comment by tymime** [comment]
Ares stopped doing it when I got a new computer with Sonoma, so I really can't give you any more details. I'm glad that you've acknowledged it as a real issue though- that's more than most coders I've encountered will admit.
